HiP-DataStore.Sdk/DataStoreService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore
+{
+    /// <summary>
+    /// A service that can be used with ASP.NET Core dependency injection.
+    /// Usage: In ConfigureServices():
+    /// <code>
+    /// services.Configure&lt;DataStoreConfig&gt;(Configuration.GetSection("Endpoints"));
+    /// services.AddSingleton&lt;DataStoreService&gt;();
+    /// </code>
+    /// </summary>
+    public class DataStoreService
+    {
+        private readonly DataStoreConfig _config;
+        private readonly ILogger<DataStoreService> _logger;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public DataStoreService(IOptions<DataStoreConfig> config, ILogger<DataStoreService> logger,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _config = config.Value;
+            _logger = logger;
+            _httpContextAccessor = httpContextAccessor;
+
+            if (string.IsNullOrWhiteSpace(config.Value.DataStoreHost))
+                logger.LogWarning($"{nameof(DataStoreConfig.DataStoreHost)} is not configured correctly!");
+        }
+
+        public ExhibitPagesClient ExhibitPages => new ExhibitPagesClient(_config.DataStoreHost)
+        {
+            Authorization = _httpContextAccessor.HttpContext.Request.Headers["Authorization"]
```

Comment:
The access to the HttpContext can be null, we should take care of that. The same for the other clients